services/cd-service/Makefile

```diff
@@ -64,11 +64,11 @@ build: bin/main
 
 build-pr:
 	echo "build on pull request"
-	$(EARTHLY) --push +build-pr --registry=$(IMAGE_REGISTRY) --tag=$(VERSION) --mirror=$(ARTIFACT_REGISTRY_MIRROR)
+	$(EARTHLY) --no-output --push +build-pr --registry=$(IMAGE_REGISTRY) --tag=$(VERSION) --mirror=$(ARTIFACT_REGISTRY_MIRROR)
```

Comment:
does this mean that the image is not pushed to the registry anymore?
That would disrupt @miguel-crespo-fdc's current workflow.

Reply:
It gets pushed, but it does not get saved locally. Check these [logs](https://github.com/freiheit-com/kuberpult/actions/runs/8754669556/job/24027084007#step:12:358)
Images are pushed, but not saved locally.